datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -404,4 +453,15 @@ public String getMappedUrl() {
         CEPH,
         AMAZON
     }
+
+
+    @SneakyThrows
+    private static String getDockerUriFromOrigUri(String uri) {
+        String dockerhost = System.getenv("DOCKER_HOST");
+        if (dockerhost == null) {
+            return uri;
+        }
+        URI dockeruri = new URI(dockerhost);
+        return "http://" + dockeruri.getHost();
```

Comment:
Better to have docker host uri and not hostname

Reply:
I do not understand.

Reply:
Docker host might be proxied too - `return "http://" + dockeruri;` should be better